scripts/globals/mobs.lua

```diff
@@ -477,7 +477,7 @@ xi.mob.onAddEffect = function(mob, target, damage, effect, params)
                 power = addBonusesAbility(mob, ae.ele, target, power, ae.bonusAbilityParams)
                 power = power * applyResistanceAddEffect(mob, target, ae.ele, 0)
                 power = adjustForTarget(target, power, ae.ele)
-                if ae.sub ~= xi.subEffect.TP_DRAIN or ae.sub ~= xi.subEffect.MP_DRAIN then
+                if ae.sub ~= xi.subEffect.TP_DRAIN and ae.sub ~= xi.subEffect.MP_DRAIN then
```

Comment:
```lua
 if ae.sub == xi.subEffect.TP_DRAIN or ae.sub == xi.subEffect.MP_DRAIN then
 ```
also makes sense. you just want it to not do the next line if its one of those 2.

Reply:
Test it and tick the box, and then won't have the problems. :+1:

Reply:
yeah i need to get lsb up and running, that's bad on me.

Reply:
loaded and tested by giving a colibri mp drain